lib/matplotlib/backend_bases.py

```diff
@@ -3224,7 +3224,16 @@ def on_tool_fig_close(e):
         return self.subplot_tool
 
     def save_figure(self, *args):
-        """Save the current figure."""
+        """
+        Save the current figure.
+
+        Returns
+        -------
+        str or `None`
+            The filepath of the saved figure.
+            For GTK4 and WebAgg backends it returns `None`.
+            Returns `None` if figure is not saved.
+        """
```

Comment:
Because we can not implement this on all backends for technical reasons (with in reason) and any backends outside of the core library (mplcairo, ipympl, the kivy backend) do not yet implement this I think it is too strong to say that returning `None` means no figure is saved, it means either no figure is saved _or_ the backend does not (yet?) implement this feature.

```suggestion
        Save the current figure.
        
        Backend implementations may choose to return
        the absolute path of the saved file, if any, as
        a string.
        
        If no file is created or the backend does not implement
        this functionality then `None` is returned.

        Returns
        -------
        str or `None`
            The filepath of the saved figure.
        """
```